package.json

```diff
@@ -1,53 +1,55 @@
 {
-    "name": "carousel",
```

Comment:
Keep the indentation as npm generates it. You don't want to reformat it every time a dependency is added.